packages/core/fs/package.json

```diff
@@ -48,6 +48,8 @@
     "check-ts": "tsc --noEmit index.d.ts"
   },
   "dependencies": {
+    "@parcel/feature-flags": "2.12.0",
+    "@parcel/watcher-watchman-js": "2.12.0",
```

Comment:
How can we make this dependency opt-in? Peer dep? try...catch around a require?

Reply:
The package is so small (less than 50KB) that I wasn't going to worry about it. I can try make it opt in though.